test/e2e/configmappropagation_default_test.go

```diff
@@ -0,0 +1,106 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"encoding/json"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+
+	"knative.dev/eventing/test/lib"
+	"knative.dev/eventing/test/lib/resources"
+)
+
+func TestDefaultConfigMapPropagation(t *testing.T) {
+	const (
+		defaultCMP        = "eventing"
+		testingCM1        = "config-testing1"
+		testingCM2        = "config-testing2"
+		eventingNamespace = "knative-eventing"
+	)
+	client := setup(t, true)
+	defer tearDown(client)
+
+	// Create two new configmaps
+	if _, err := client.Kube.Kube.CoreV1().ConfigMaps(eventingNamespace).Create(resources.ConfigMap(testingCM1, map[string]string{
+		"firstdata":  "data1",
+		"seconddata": "data2",
+	})); err != nil {
+		t.Fatalf("Failed to create configmap "+testingCM1+": %v", err)
+	}
+
+	if _, err := client.Kube.Kube.CoreV1().ConfigMaps(eventingNamespace).Create(resources.ConfigMap(testingCM2, map[string]string{
+		"thirddata":  "data3",
+		"fourthdata": "data4",
+	})); err != nil {
+		t.Fatalf("Failed to create configmap "+testingCM2+": %v", err)
+	}
+
+	defer deleteConfigMap(client, eventingNamespace, testingCM1, testingCM2)
```

Comment:
In `CreateConfigMapOrFail` if you add the CM in the tracker like https://github.com/knative/eventing/blob/9d4870e64543251f04221f279f0cec0fbae9dcc3/test/lib/creation.go#L55, this function can be deleted as tracker will delete all resources after the test is finished.